src/main/java/com/bandwidth/sqs/consumer/SqsConsumer.java

```diff
@@ -82,6 +84,7 @@ public SqsConsumer(SqsConsumerBuilder<T> builder) {
         this.expirationStrategy = requireNonNull(builder.expirationStrategy);
 
         this.sqsQueue = builder.sqsQueue;
+        this.queueAttributes = sqsQueue.getAttributes().blockingGet();
```

Comment:
Hmm, it would be nice if there wasn't blocking here... but I guess to avoid it you would have to make it return a `Single<SqsConsumer>` or something